Dockerfile

```diff
@@ -0,0 +1,30 @@
+
+FROM alpine:latest
+
+WORKDIR /app/test
+
+
+RUN apk update && apk add --no-cache \
+    curl \
+    && GO_VERSION=$(curl -sL https://go.dev/VERSION?m=text | head -n 1) \
+    && wget https://go.dev/dl/${GO_VERSION}.linux-amd64.tar.gz \
+    && tar -C /usr/local -xzf ${GO_VERSION}.linux-amd64.tar.gz \
+    && rm ${GO_VERSION}.linux-amd64.tar.gz \
+    && curl -sL https://taskfile.dev/install.sh | sh -s -- -b /usr/local/bin \
```

Comment:
¿Por qué usas e instalas `curl` si puedes usar wget?
De veras, si copiáis las líneas de algún lado, el que sea, tratad de enteraros por lo menos de qué se tratan...